ui/appui-react/src/appui-react/statusfields/SnapMode.tsx

```diff
@@ -78,6 +91,9 @@ const getSnapModeIconNameFromMode = (
 /** Defines properties supported by the SnapMode Field Component. */
 // eslint-disable-next-line deprecation/deprecation
 interface SnapModeFieldProps extends CommonProps {
+  /** The list of snap modes displayed in the status bar. Default to all the modes.
+   */
+  snapModes?: SnapMode[];
```

Comment:
The current property `snapMode` seems to already support multiple active snap modes via the enum flags. Although the PR description implies that it is something new we are adding?

This new property seems to allow filtering of available snap modes. Should we rename it to `availableSnapModes`?

Reply:
It was supported, but the UI wasn't allowing it. 
The SnapModeField items acted like radio buttons, now they act like checkboxes. 

Agreed, I will rename it.

Reply:
Could you clarify what "UI wasn't allowing it" means? Wasn't it rendered correctly or only the `onChange` handler would not do the bit-wise operations?

Reply:
The onChange would not do the bit-wise operation, so it acted like a radio button. 
In other words, when clicking a mode, it toggled OFF the other mode, so only one mode was ON.